data/dbus/snapd.conf

```diff
@@ -0,0 +1,3 @@
+<busconfig>
```

Comment:
This will not work with re-exec, i.e. this config file needs to be in the regular deb package right now and if snapd just re-execs it won't be there. So we probably need to add code that writes it if its missing. Which has the downside that it will also mean that we leave files behind when snapd gets removed.